lxd-agent/shared/api.go

```diff
@@ -0,0 +1,25 @@
+package shared
```

Comment:
I thought we said we'd just use the same version and list of extensions as LXD itself?

Can you get rid of this and just use version.APIVersion and version.APIExtensions?

Reply:
Done